cmd/kepctl/cmd/create.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"k8s.io/enhancements/pkg/kepctl"
+)
+
+// TODO: Struct literal instead?
+var createOpts = kepctl.CreateOpts{}
+
+var createCmd = &cobra.Command{
+	Use:           "create [KEP]",
+	Short:         "Create a new KEP",
+	Long:          "Create a new KEP using the current KEP template for the given type",
+	Example:       `  kepctl create sig-architecture/000-mykep`,
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return createOpts.Validate(args)
+	},
+	RunE: func(*cobra.Command, []string) error {
+		return runCreate(createOpts)
+	},
+}
+
+func init() {
```

Comment:
The use of init means no one can cherry pick this command to remix their own cli. I prefer explicitly adding commands and in this way: https://github.com/kubernetes-sigs/zeitgeist/blob/master/buoy/commands/commands.go